compiler/rustc_error_codes/src/error_codes/E0015.md

```diff
@@ -1,33 +1,26 @@
-A constant item was initialized with something that is not a constant
-expression.
+A non-`const` function was called in a `const` context.
 
-Erroneous code example:
+Example of erroneous code:
```

Comment:
Based on the other code explanations (like [this one](https://github.com/rust-lang/rust/blob/master/compiler/rustc_error_codes/src/error_codes/E0005.md)), the actual correct sentence is "Erroneous code example").

Reply:
Done, thanks.